# Result of the Euclidean Cluster Extraction for segmenting objects
Header 	              header
geometry_msgs/Point[] cluster_centroids
float32[]             cluster_sizes
